mysql-test/suite/rocksdb/t/autoinc_vars_thread.test

```diff
@@ -0,0 +1,53 @@
+--source include/have_rocksdb.inc
```

Comment:
Test only needs MyRocks, not as default, properly uses ENGINE=ROCKSDB where necessary.